src/components/IncorporationAgreement/AgreementType.vue

```diff
@@ -102,6 +139,18 @@ export default class AgreementType extends Vue {
         .summaryDescription
     } else { return '' }
   }
+
+  @Emit('haveChanges')
```

Comment:
Can we change this to saving it in the store instead?
Also we will need a getter for App to use to display/hide the Fee Summary. (Similar getter to `isApplicationValid` I think?)

Reply:
Looked into this. I think we will have changes to the company, agreement type, correction and App components for this. I will either createa a follow up PR once Cameron's PR is in. It will be easy to do all that together.

Reply:
OK.
I looked into also and there are 2 things to think about:
1. There is a `haveChanges` store property to keep track of whether the user has done any changes (to prevent their exit if they have unsaved changes). I think the component-has-changed flags can replace this.
2. App will have to check whether there are any correction changes or alteration changes or ??? when deciding whether to display the Fee Summary. This is probably a simple getter with A || B || C but it doesn't exist yet.